app/assets/javascripts/discourse/app/controllers/user-activity.js

```diff
@@ -35,6 +35,18 @@ export default Controller.extend({
       : I18n.t("drafts.label");
   },
 
+  @discourseComputed("currentUser.pending_posts_count")
+  pendingLabel(count) {
+    return count > 0
+      ? I18n.t("pending_posts.label_with_count", { count })
+      : I18n.t("pending_posts.label");
+  },
+
+  @discourseComputed("currentUser.pending_posts_count")
```

Comment:
Rather than making a CP that you only use once, you can use `{{#if (gt currentUser.pending_posts_count 0)}}` in your template.

Reply:
I’m used to avoid putting logic directly in the views but if having a CP used only once is a smell then I’ll change it :wink: